uimaj-core/src/main/java/org/apache/uima/util/CasCreationUtils.java

```diff
@@ -91,6 +91,18 @@
   
   private final static FeatureDescription[] EMPTY_FEAT_DESC_ARRAY = new FeatureDescription[0];
   
+  /**
+   * Creates a new CAS instance.
```

Comment:
Is this just for testing?  If so, can the comment say this?

Perhaps the comment could also say it creates a CAS with no type system, no priorities, and no indexes, with defaults for other properties?

Reply:
I actually use such a method regularly elsewhere and found it again to be useful here. No - not only for testing.

Reply:
ok.